sdk/core/core-http/lib/util/xml.browser.ts

```diff
@@ -99,9 +115,15 @@ function domToObject(node: Node): any {
 
 const serializer = new XMLSerializer();
 
-export function stringifyXML(obj: any, opts?: { rootName?: string }) {
+export function stringifyXML(content: any, opts?: { rootName?: string }): string {
+  if (content == undefined) {
+    throw new Error("Cannot convert undefined or null to object");
+  }
+  if (content == "") {
```

Comment:
Should this be `===`?

Reply:
No since == has type conversion involved that's useful for `string` vs `String` checks